acra-censor/common/logging_logic_test.go

```diff
@@ -16,7 +16,7 @@ const testSerializationTime = 100 * time.Millisecond
 
 // sleepTime provide extra time to serialize in background goroutine before check
 // default value that will be changed for tests in init function
-const sleepTime = testSerializationTime * 2
+const sleepTime = testSerializationTime * 3
```

Comment:
👀

Reply:
actually, it's a weird situation. it works fine on our acra-ce repo (here) and often fail on acra-ee that runs the same tests ) so, I guess it depends on resource shaping for private repositories when you use free credits. but it's just my opinion...

Reply:
yeah, might be